internal/envoy/v3/cluster.go

```diff
@@ -198,6 +198,22 @@ func ExtensionCluster(ext *dag.ExtensionCluster) *envoy_config_cluster_v3.Cluste
 	}
 	cluster.TypedExtensionProtocolOptions = protocolOptions(http2Version, ext.ClusterTimeoutPolicy.IdleConnectionTimeout, nil)
 
+	if envoy.AnyPositive(ext.CircuitBreakersSettings.MaxConnections, ext.CircuitBreakersSettings.MaxPendingRequests, ext.CircuitBreakersSettings.MaxRequests, ext.CircuitBreakersSettings.MaxRetries, ext.CircuitBreakersSettings.PerHostMaxConnections) {
```

Comment:
nit: can we try to unify this and the above `Cluster()` method logic into a helper, something like `applyCircuitBreakerSettings()`

Reply:
fixed in https://github.com/projectcontour/contour/pull/6539/commits/4790f07b4d94965415cb8ef2ac8004949da334e1